sig-security-tooling/cve-feed/hack/fetch-official-cve-feed.py

```diff
@@ -91,10 +91,20 @@ def getCVEStatus(state, state_reason):
     if len(title) > 0:
         cve['summary'] = title[-1]
         if len(title) > 1:
-            cve_id = title[0]
-            cve['id'] = cve_id
-            cve['external_url'] = f'https://www.cve.org/cverecord?id={cve_id}'
-            cve['_kubernetes_io']['google_group_url'] = f'https://groups.google.com/g/kubernetes-announce/search?q={cve_id}'
+            cve_ids = title[0].split(', ')
+            first_cve_id = cve_ids[0]
+            cve['id'] = first_cve_id
+            cve['external_url'] = f'https://www.cve.org/cverecord?id={first_cve_id}'
+            cve['_kubernetes_io']['google_group_url'] = f'https://groups.google.com/g/kubernetes-announce/search?q={first_cve_id}'
+
+            # Add additional entries for any remaining CVE IDs
+            for additional_cve_id in cve_ids[1:]:
+                additional_cve = cve.copy()
+                additional_cve['id'] = additional_cve_id
+                additional_cve['external_url'] = f'https://www.cve.org/cverecord?id={additional_cve_id}'
+                additional_cve['_kubernetes_io'] = cve['_kubernetes_io'].copy()
```

Comment:
Is this needed, since we copy `cve` dict in its entirety already?

Reply:
I believe this is needed, the copy command makes a shallow copy. Without shallow copying the nested dictionary, the command to update the url updates the keys in both dictionaries. I've replaced both copy commands with a single deepcopy for brevity :+1: